rust/processor/src/db/common/models/emojicoin_models/models/market_registration_event.rs

```diff
@@ -25,6 +25,7 @@ pub struct MarketRegistrationEventModel {
     pub bump_time: chrono::NaiveDateTime,
     pub market_nonce: i64,
     pub trigger: enums::Trigger,
+    pub market_address: String,
```

Comment:
Per https://github.com/econia-labs/aptos-indexer-processors/pull/53#pullrequestreview-2310994448, while yes this field can technically be derived, I foresee that we will alleviate downstream indexing issues later if we simply include it here, in all applicable contexts

- [ ] Include in all other tables as needed for consistency and ease of indexing